tests/helpers/welcome-steps.js

```diff
@@ -0,0 +1,19 @@
+import { registerAsyncHelper } from '@ember/test';
```

Comment:
This is pretty cool. I didn't realize you can automate the login test process so elegantly. These tests would be really nice for volume account creation testing and other things that involve heavy traffic.

Reply:
@bakerjoh000 About allowing tests to access gatekeeper routes... I made a test helper in the gatekeeper-stubber.js file that injects a dummy gatekeeper service into the routes.. You can call the methods from that helper to open up gatekeeper.